manifests/lcm.pp

```diff
@@ -384,6 +384,21 @@
     before   => Class['::foreman'],
   }  
 
+  package { 'libkadm5':
+    ensure   => installed,
+  }
+  
+  package { 'rubygem-rkerberos':
```

Comment:
Maybe add a comment why this is needed? This is because of the EPEL repository issue, right?